src/NativeShell/nativeshell.js

```diff
@@ -0,0 +1,36 @@
+function NativeShell() {}
+
+NativeShell.prototype.getDeviceInformation = function(successCallback, errorCallback) {
+  cordova.exec(successCallback, errorCallback, 'NativeShell', 'getDeviceInformation', []);
+}
+
+NativeShell.prototype.enableFullscreen = function(successCallback, errorCallback) {
+  cordova.exec(successCallback, errorCallback, 'NativeShell', 'enableFullscreen', []);
+}
+
+NativeShell.prototype.disableFullscreen = function(successCallback, errorCallback) {
+  cordova.exec(successCallback, errorCallback, 'NativeShell', 'disableFulsceeen', []);
+}
+
+NativeShell.prototype.updateMediaSession = function(successCallback, errorCallback, action, isLocalPlayer, itemId, title, artist, album, duration, position, imageUrl, canSeek, isPaused) {
```

Comment:
can we wrap long lines like this?

Reply:
Sure, I was actually going to just pass the variables directly anyway, which would fix the issue.